packages/account/src/Components/forms/personal-details-form.jsx

```diff
@@ -75,7 +75,7 @@ const PersonalDetailsForm = props => {
     };
 
     const getFieldHint = field_name =>
-        is_qualified_for_idv || is_rendered_for_onfido ? (
+        (is_svg && !is_mf) || is_rendered_for_onfido ? (
```

Comment:
cant we use is_svg_only here as the conditions are same
 const is_svg_only = is_svg && !is_mf;

Reply:
sure @amina-deriv 
Will update